Test-Rack-Software/util/dynamic_url.py

```diff
@@ -0,0 +1,20 @@
+import requests
+
+api_source = "https://raw.githubusercontent.com/ISSUIUC/ISS_HILSIM/active_server_url/server_url.txt"
+api_default = "http://localhost/"
+
+def get_dynamic_url(kamaji_target="main") -> str:
+    """Retrieves the api url for kamaji
+    @kamaji_target: Which target to retrieve the URI for? `main` will always be the main Kamaji server."""
+    if api_source is not None:
+        print("(dynamic url) Retrieving dynamic url")
+        result = requests.get(api_source)
+        try:
+            print("(dynamic url) Successfully retrieved dynamic url " + str(result.content.decode().strip()))
+            return result.content.decode().strip()
+        except Exception as e:
+            print("Failed to retrieve dynamic url:", e)
+            return api_default.strip()
```

Comment:
Remove strip

Reply:
why remove it? don't we want to remove any erroneous whitespace?